services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/team/TeamManagerImpl.java

```diff
@@ -437,8 +439,12 @@ public PaginatedTeamIds listIdsByMember(String teamMemberId, String nextPageToke
 	 * @see org.sagebionetworks.repo.manager.team.TeamManager#get(java.lang.String)
 	 */
 	@Override
-	public Team get(String id) throws DatastoreException, NotFoundException {
-		return teamDAO.get(id);
+	public Team get(String teamId) throws DatastoreException, NotFoundException {
+		try {
+			return teamDAO.get(teamId);
+		} catch (NotFoundException e) {
+			throw new NotFoundException("Team does not exist for teamId: " + teamId ,e);
```

Comment:
If we want to use a more friendly message, it makes sense to do this at the DAO so that the message is coherent wherever it is used.